apps/hexagon_api/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(hexagon_api)
+
+include(ExternalProject)
+
+# Required variables:
+#   ANDROID_ABI
+#   ANDROID_PLATFORM
+#   USE_ANDROID_TOOLCHAIN (Android toolchain .cmake file)
+#   USE_HEXAGON_ARCH
+#   USE_HEXAGON_SDK
+#   USE_HEXAGON_TOOLCHAIN (Path to Hexagon toolchain ending with "Tools")
+
+set(TVM_SOURCE_DIR "${CMAKE_SOURCE_DIR}/../..")
+set(HEXAGON_API_BINARY_DIR "${CMAKE_BINARY_DIR}/hexagon_rpc")
```

Comment:
change HEXAGON_API_BINARY_DIR to this to be compatible with current HexagonLauncher design:
set(HEXAGON_API_BINARY_DIR "${CMAKE_BINARY_DIR}/../../../build/hexagon_rpc")

This is with the assumption of building this CmakeLists.txt under `apps/hexagon_api/build`. Not the best solution, but we need to make sure everything is copied to `build/hexagon_rpc`.

Reply:
This is going to be really fragile.  Instead I propose a flag `USE_OUTPUT_BINARY_DIR` that could be set to any location, including `.../build/hexagon_rpc`.

Reply:
@kparzysz-quic makes sense. Thanks!